portfolio/src/main/java/com/google/sps/servlets/COVID19DataServlet.java

```diff
@@ -0,0 +1,45 @@
+package com.google.sps.servlets;
+
+import com.google.gson.Gson;
+import java.io.IOException;
+import java.util.LinkedHashMap;
+import java.util.Scanner;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+/** When requested, returns the covid-19 active cases in each country as a JSON object */
+@WebServlet("/covid-19-data")
+public class COVID19DataServlet extends HttpServlet {
+
+  @Override
+  /** Reads the data from the local file */
+  public void init() {
+    Scanner scanner = new Scanner(getServletContext().getResourceAsStream(
+      "/WEB-INF/country_wise_latest.csv"));
+    scanner.nextLine();
```

Comment:
Add comments so others could know you want to skip the header.

Reply:
Added!

Reply:
Actually you want to skip the **header** line.